semconv/src/main/java/io/opentelemetry/semconv/trace/attributes/SemanticAttributes.java

```diff
@@ -579,8 +681,18 @@ public final class SemanticAttributes {
    */
   public static final AttributeKey<String> HTTP_URL = stringKey("http.url");
 
-  /** The ordinal number of request re-sending attempt. */
-  public static final AttributeKey<Long> HTTP_RETRY_COUNT = longKey("http.retry_count");
```

Comment:
We decided to not delete, and instead `@deprecate` correct?

Reply:
fixed